calyx/src/passes/merge_static_par.rs

```diff
@@ -0,0 +1,124 @@
+use crate::ir::Attributes;
+use crate::ir::Enable;
+use crate::ir::{
+    self,
+    traversal::{Action, Named, VisResult, Visitor},
+    RRC,
+};
+use linked_hash_map::LinkedHashMap;
+use std::iter::Iterator;
+use std::mem;
+use std::rc::Rc;
+
+/// Pass to do something
```

Comment:
Add a comment explaining what the pass does. See example here: https://github.com/cucapra/calyx/blob/master/calyx/src/passes/collapse_control.rs#L5